<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playGroundFrame</class>
 <widget class="QFrame" name="playGroundFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #610094;</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>581</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_spielfeld">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_3_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 0px;
margin: 0px;
border: 0px solid black;


background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #3E065F, stop: 0.4
                                  #700B97, stop:1 #8E05C2);

font-size: 40pt;
border-radius: 20px;
color: #F6F6F6;</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_1_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 0px;
margin: 0px;
border: 0px solid black;


background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #8E05C2, stop: 0.4
                                  #700B97, stop:1 #8E05C2);

font-size: 40pt;
border-radius: 20px;
color: #F6F6F6;</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_1_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 0px;
margin: 0px;
border: 0px solid black;


background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #8E05C2, stop: 0.4
                                  #700B97, stop:1 #3E065F);

font-size: 40pt;
border-radius: 20px;
color: #F6F6F6;</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_2_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 0px;
margin: 0px;
border: 0px solid black;


background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #3E065F, stop: 0.4
                                  #700B97, stop:1 #8E05C2);

font-size: 40pt;
border-radius: 20px;
color: #F6F6F6;</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_3_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 0px;
margin: 0px;
border: 0px solid black;


background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #8E05C2, stop: 0.4
                                  #700B97, stop:1 #8E05C2);

font-size: 40pt;
border-radius: 20px;
color: #F6F6F6;</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_3_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 0px;
margin: 0px;
border: 0px solid black;


background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #8E05C2, stop: 0.4
                                  #700B97, stop:1 #3E065F);

font-size: 40pt;
border-radius: 20px;
color: #F6F6F6;</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_2_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 0px;
margin: 0px;
border: 0px solid black;


background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #8E05C2, stop: 0.4
                                  #700B97, stop:1 #3E065F);

font-size: 40pt;
border-radius: 20px;
color: #F6F6F6;</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_2_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 0px;
margin: 0px;
border: 0px solid black;


background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #8E05C2, stop: 0.4
                                  #700B97, stop:1 #8E05C2);

font-size: 40pt;
border-radius: 20px;
color: #F6F6F6;</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_1_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 0px;
margin: 0px;
border: 0px solid black;
color: #F6F6F6;

background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #3E065F, stop: 0.4
                                  #700B97, stop:1 #8E05C2);

font-size: 40pt;
border-radius: 20px</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_x">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 0px;
margin: 0px;
border: 0px solid black;
color: #F6F6F6;

font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>You : X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_o">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 0px;
margin: 0px;
border: 0px solid black;
color: #F6F6F6;

font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>O : Computer</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
